qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -130,9 +185,13 @@ def load_account(self) -> Optional[AccountProvider]:
                 credentials are found.
             IBMQAccountCredentialsInvalidUrl: If invalid IBM Quantum Experience
                 credentials are found.
+            IBMQAccountError: If the default provider stored on disk could not be found.
+            HubGroupProjectInvalidStateError: If the default provider stored on disk could
```

Comment:
`HubGroupProjectInvalidStateError` should probably be converted to an `IBMQAccountError`, if that's the high level exception class used in this module.